viam_cartographer.go

```diff
@@ -237,6 +253,7 @@ func New(
 		Named:                         c.ResourceName().AsNamed(),
 		lidar:                         timedLidar,
 		imu:                           timedIMU,
```

Comment:
[q] will we be storing imu and odom separately?

Reply:
Yes. The odometer will be handled in the next PR.